nix/workbench/run.sh

```diff
@@ -709,12 +703,12 @@ legacy_run_manifest() {
       --arg Node          $node
       --arg NodeBranch    $node_branch
       --arg NodeApproxVer $node_ver
-      --arg Network  $(node_cabal_source_at $node ouroboros-network)
-      --arg Ledger   $(node_cabal_source_at $node cardano-ledger)
-      --arg Plutus   $(node_cabal_source_at $node plutus)
-      --arg Crypto   $(node_cabal_source_at $node cardano-crypto)
-      --arg Base     $(node_cabal_source_at $node cardano-base)
-      --arg Prelude  $(node_cabal_source_at $node cardano-prelude)
+      --arg Network       $(node_cabal_source_at $node ouroboros-network)
+      --arg Ledger        $(node_cabal_source_at $node cardano-ledger)
+      --arg Plutus        $(node_cabal_source_at $node plutus)
+      --arg Crypto        $(node_cabal_source_at $node cardano-crypto)
+      --arg Base          $(node_cabal_source_at $node cardano-base)
+      --arg Prelude       $(node_cabal_source_at $node cardano-prelude)
```

Comment:
Can we avoid this (seemingly) unnecessary change?

Reply:
What are the formatting rules for bash scripts in this repository?

Reply:
I reverted this change.